studio/components/ui/ProjectSettings/DisplayApiSettings.tsx

```diff
@@ -44,7 +44,14 @@ const DisplayApiSettings = () => {
           <p className="text-sm text-scale-1000">
             Your API is secured behind an API gateway which requires an API Key for every request.
             <br />
-            You can use the keys below to use Supabase client libraries.
+            You can use the keys below in the Supabase client libraries.
+            <br />
+            <a href="https://supabase.com/docs#client-libraries" target="_blank" rel="noreferrer">
+              <Button icon={<IconBookOpen />} type="default" className="mt-4">
+                Client Docs
+              </Button>
+            </a>
+            .
```

Comment:
Extra .
![image](https://github.com/supabase/supabase/assets/19742402/c77e1769-7eb8-416d-ada6-84974ca4fde1)

Reply:
how on earth did you spot that!